### PR TITLE
Add organization management documentation

### PR DESCRIPTION
**Pull request by AnuradhaSK**
## Purpose
Part of https://github.com/wso2/product-is/issues/16548

- [x] Updted doc structure for organization management
Screenshot 2023-11-10 at 17 26 31

- [x] Updated `Set up organizations` doc
- [x] Updated `Onboard organization administrators` doc
- [x] Updated `Administration of organizations` doc

TODO:

- [ ] Fix broken links for API endpoints
- [ ] Define organization (root) in About WSO2 Identity Server doc

### CODE REVIEW

File: en/identity-server/next/docs/guides/organization-management/manage-organizations.md [hunk 1]
```diff
@@ -1,2 +1,101 @@
...
+To create organizations:
+
+- You need to have organization management privileges in the organization (root).
+
+## Create an organization
+
+Follow the steps given below to create an organization.
+
+1. First, login to your organization (root) from the {{ product_name }} Console.
+2. On the {{ product_name }} Console, go to **Organizations** under **Organization Management** and click **Add Organization**.
```
Comment (divyaamunugama) [inline en/identity-server/next/docs/guides/organization-management/manage-organizations.md:22]:
```suggestion
2. On the {{ product_name }} Console, go to **Organization Management** > **Organizations** and click **Add Organization**.
```

File: en/identity-server/next/docs/guides/organization-management/manage-organizations.md [hunk 2]
```diff
@@ -1,2 +1,101 @@
...
+3. In the **Add Organization** dialog box that opens, enter a name and description for your organization.
+
+    ![Create organization]({{base_path}}/assets/img/guides/organization/manage-organizations/add-organization.png){: width="500" style="display: block; margin: 0 auto; border: 0.3px solid lightgrey;"}
+
+4. Click **Register** to add the new organization. You will now see your organization listed as below.
+
+    ![The list of organizations]({{base_path}}/assets/img/guides/organization/manage-organizations/organization-list.png){: width="700" style="display: block; margin: 0 auto; border: 0.3px solid lightgrey;"}
+
+5. Click the **Switch** icon next to the organization to go to the organization console.
```
Comment (divyaamunugama) [inline en/identity-server/next/docs/guides/organization-management/manage-organizations.md:31]:
```suggestion
5. Click **Switch** next to the organization to switch to the new organization's console.
```

File: en/identity-server/next/docs/guides/organization-management/manage-organizations.md [hunk 3]
```diff
@@ -1,2 +1,101 @@
...
+    - Configure the branding of the organization. Learn more about [branding]({{base_path}}/guides/branding/configure-ui-branding/).
+    - Create organizations.
+
+## Add organization attributes
+
+{{ product_name }} lets you maintain additional data about the organization.
+
+For example, you can add information about the organization's location, country, number of employees, etc.
+
+To add additional information about your organizations:
```
Comment (divyaamunugama) [inline en/identity-server/next/docs/guides/organization-management/manage-organizations.md:50]:
```suggestion
To add additional information about your organization:
```

Comment (divyaamunugama) [reply en/identity-server/next/docs/guides/organization-management/manage-organizations.md:50]: Shouldn't this be to add additional attributes? the remaining section explain on how to add additional attributes

File: en/identity-server/next/docs/guides/organization-management/manage-organizations.md [hunk 4]
```diff
@@ -1,2 +1,101 @@
...
+5. Click **Update** to save the configurations.
+
+## Switch between organizations
+
+When you have multiple [organizations]({{base_path}}/guides/organization-management/manage-organizations/) for your primary business organization, you can switch between them on the {{ product_name }} Console by selecting the required organization from the list as shown below.
+
+![organizations]({{base_path}}/assets/img/guides/organization/manage-organizations/organizations.png){: width="700" style="display: block; margin: 0 auto; border: 0.3px solid lightgrey;"}
+
+
+## Create an organization for your organization
```
Comment (divyaamunugama) [inline en/identity-server/next/docs/guides/organization-management/manage-organizations.md:68]: is the meaning of this topic ....

Create a new organization for the root organization?

File: en/identity-server/next/docs/guides/organization-management/manage-root-organizations.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+!!! note 
+    If you have a B2B platform, this will also be the organization (root) of your platform. Your organization (root) will work with other businesses to make your applications available to their users. Such businesses should be defined as [organizations]({{base_path}}/guides/organization-management/manage-suborganizations/) in your organization (root).
+
+The following guides explain how you can create and manage these organizations in {{ product_name }}.
+
+## Create a new organization (root)
+
+To create a new organization (root) in {{ product_name }}:
+
+1.
```
Comment (divyaamunugama) [inline en/identity-server/next/docs/guides/organization-management/manage-root-organizations.md:15]: This section is missing

File: en/identity-server/next/docs/guides/organization-management/manage-root-organizations.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+To create a new organization (root) in {{ product_name }}:
+
+1. 
+
+!!! note
+    If you a have B2B platform, you can now onboard your organizations. See the instructions on [setting up organizations]({{base_path}}/guides/organization-management/manage-organizations/) and build your B2B organization structure.
+
+## Delete organization (root)
+
+You cannot remove organization (root) using the {{ product_name }} Console.
```
Comment (divyaamunugama) [inline en/identity-server/next/docs/guides/organization-management/manage-root-organizations.md:22]:
```suggestion
You cannot remove the organization (root) using the {{ product_name }} Console.
```

File: en/identity-server/next/docs/guides/organization-management/manage-root-organizations.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+## What's next?
+- Learn about managing administrators and users of an organization (root) in the [user management]({{base_path}}/guides/users/) section.
```
Comment (divyaamunugama) [inline en/identity-server/next/docs/guides/organization-management/manage-root-organizations.md:25]:
```suggestion
- Learn about [managing administrators and users]({{base_path}}/guides/users/) of an organization (root).
```

File: en/identity-server/next/docs/guides/organization-management/manage-root-organizations.md (hunk 6 above)
Comment (AnuradhaSK) [reply en/identity-server/next/docs/guides/organization-management/manage-root-organizations.md:22]: This doc is not used now. Wii remove

File: en/identity-server/next/docs/guides/organization-management/manage-root-organizations.md (hunk 7 above)
Comment (AnuradhaSK) [reply en/identity-server/next/docs/guides/organization-management/manage-root-organizations.md:25]: This doc is not used now. Wii remove

File: en/identity-server/next/docs/guides/organization-management/manage-root-organizations.md (hunk 5 above)
Comment (AnuradhaSK) [reply en/identity-server/next/docs/guides/organization-management/manage-root-organizations.md:15]: This doc is not used now. Will remove

**Review by divyaamunugama — APPROVED** [review]

**Pull request merged**